components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -8384,17 +8389,22 @@ def setActiveChannels(self, channels, windows=None, colors=None,
                             Must be list for each channel
         :param colors:      List of colors. ['F00', None, '00FF00'].
                             Must be item for each channel
-        :param reverseMaps: List of boolean (or None). If True/False then
+        :param invertMaps:  List of boolean (or None). If True/False then
                             set/remove reverseIntensityMap on channel
         """
+        if reverseMaps is not None:
```

Comment:
If this is set and invertMaps is _not_ set, then invertMaps should be updated to make use of this value (even if the warning should still be printed)

Reply:
Fixed in 23fbd02ac69e342a775fb81545867e06b164d2d3